### PR TITLE
[WIP] Features/zones list buttons

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/y2firewall/widgets/allowed_services.rb [hunk 1]
```diff
@@ -93,10 +93,14 @@ def help
...
       def validate
+        # check for missed selection in ncurses only as in qt selected entries is basically
+        # just highlight and after move it is still selected, so more confusing.
+        return true unless Yast::UI.TextMode
+
         return true if selected_services.empty?
         # TRANSLATORS: popup question
-        msg = _("The selection of services will be lost if you leave without\n" \
-          "applying the changes.\n\nDo you really want to continue?\n")
+        msg = _("The selection of services will be reset lost if you leave page\n" \
```
Comment (teclator) [inline src/lib/y2firewall/widgets/allowed_services.rb]:
```ruby
_("The selection of services will be lost if you leave the page\n" \
    "without moving them with Add/Remove.\n\nDo you really want to continue?\n") 
```

File: src/lib/y2firewall/widgets/overview.rb [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+require "cwm/tree"
+require "cwm/tree_pager"
+require "y2firewall/widgets/pages"
+require "y2firewall/helpers/interfaces"
+
+module Y2Firewall
+  module Widgets
+    # A tree that is told what its items are.
+    # We need a tree whose items include Pages that point to the OverviewTreePager.
+    class OverviewTree < CWM::Tree
```
Comment (teclator) [inline src/lib/y2firewall/widgets/overview.rb]: Is this needed? I think we already added it

Comment (jreidinger) [reply src/lib/y2firewall/widgets/overview.rb]: added where? to CWM?

**Comment by jreidinger** [comment]
test failure is due to missing yast2 dependency

File: package/yast2-firewall.changes [hunk 3]
```diff
@@ -1,3 +1,12 @@
+-------------------------------------------------------------------
+Fri Nov 30 15:13:02 UTC 2018 - jreidinger@suse.com
+
+- Add ability to add/edit/remove zone (fate#324662)
+- Show warning about lost selection only in ncurses and improve
+  text
```
Comment (teclator) [inline package/yast2-firewall.changes]: Looks ugly :P, please remove the extra line and rebase for fixing conflicts

**Pull request merged**